scenarios/sample-download.xml

```diff
@@ -32,8 +32,8 @@
                 <class>com.sonatype.nexus.perftest.maven.DownloadOperation</class>
                 <repo>public</repo>
                 <paths>
-                  <class>com.sonatype.nexus.perftest.maven.HttpdLogParser</class>
-                  <logfile>data/maven-3.1-build-artifact-access.log.gz</logfile>
+                  <class>com.sonatype.nexus.perftest.maven.CsvLogParser</class>
+                  <logfile>data/maven-3.3.4-build-artifact-access.csv.gz</logfile>
```

Comment:
3.3.4 is/was just a recently tagged version of maven. It's not important that it be that version in particular, but I thought I'd note the version number in the data file name as an older maven-build artifact-request script was rife with 404s.